www/klei00/cv06/database/CategoriesDB.php

```diff
@@ -0,0 +1,25 @@
+<?php require_once __DIR__.'/Database.php'; ?>
+
+<?php
+
+class CategoriesDB extends Database {
+    protected $tableName = 'categories';
+
+    public function fetchAll(){
+        $sql = 'SELECT * FROM '.$this->tableName;
```

Comment:
Ahoj, tahle metoda fetchAll vypadá úplně stejně u všech tabulek. Šlo by ji dát do třídy Database, ať se neopakuje v těch podtřídách? Jinak pěkné.

Reply:
No jo, to mi nějak uniklo. Opraveno.